src/patternfly/components/Wizard/docs/code.md

```diff
@@ -0,0 +1,43 @@
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-expanded="true"` | `.pf-c-wizard__toggle` | Indicates that the steps menu is visible. **Required** |
+| `aria-expanded="false"` | `.pf-c-wizard__toggle` | Indicates that the steps menu is hidden. **Required** |
+| `aria-controls=[ID of wizard nav]` | `.pf-c-wizard__toggle` | Identifies the element controlled by the toggle. **Required** |
+| `aria-haspopup="true"` | `.pf-c-wizard__toggle` | Indicates that the toggle button has a popup menu. **Required** |
+| `aria-label="close"` | `.pf-c-wizard__toggle-icon` | Gives the close button an accessible name. **Required** |
+| `aria-label="Steps"` | `.pf-c-wizard__nav` | Gives the steps nav element an accessible name. **Required** |
+| `aria-hidden="true"` | `.pf-c-wizard__toggle-icon`, `.pf-c-wizard__toggle-divider` | Hides the icon from assistive technologies. |
+| `aria-disabled="true"` | `.pf-c-wizard__nav-link` | Indicatese that the element is disabled. **Required for disabled links with `.pf-m-disabled`** |
+| `tabindex="-1"` | `.pf-c-wizard__nav-link` | Removes a link from keyboard focus. **Required for disabled links with `.pf-m-disabled`** |
+
+## Usage
+
+| Class | Applied To | Outcome |
+| -- | -- | -- |
+| `.pf-c-wizard` | `<div>` | Initiates the wizard component. |
+| `.pf-c-wizard__header` | `<header>` | Initiates the header . |
+| `.pf-c-wizard__close` | `.pf-c-button.pf-m-plain` | Initiates the close button. |
```

Comment:
We can, but per the [BEM guidelines](https://cssguidelin.es/#more-layers)

> If we were to add another Element—called, let’s say, .person__eye {}—to this .person {} component, we would not need to step through every layer of the DOM. That is to say, the correct notation would be .person__eye {}, and not .person__head__eye {}. Your classes do not reflect the full paper-trail of the DOM.

If there were more than 1 description, close, and title, it would make sense to use `header-[element]`, but in this case there aren't. Even then, I think we could add `.pf-c-wizard__footer-description` and leave `.pf-c-wizard__description` and it wouldn't be confusing. Some of our classes get unnecessarily long :) But I'm fine with it either way. What do you think, still want to change it?

Reply:
Oh I never knew this I thought it was a hard rule. I'm definitely fine with leaving this!